config/settings/local.py

```diff
@@ -13,11 +13,8 @@
     default="aHdCBMHXuxIxEhfRGFRp7Cp3N9CqEZEEAvwZVlBCazKExkEnzvVs4bYWC8Qqh9lg",
 )
 # https://docs.djangoproject.com/en/dev/ref/settings/#allowed-hosts
-ALLOWED_HOSTS = [
-    "localhost",
-    "0.0.0.0",
-    "127.0.0.1",
-]
+allowed_hosts = os.getenv("DJANGO_ALLOWED_HOSTS", "localhost,0.0.0.0,127.0.0.1,*")
+ALLOWED_HOSTS = [allowed_host.strip() for allowed_host in allowed_hosts.split(",")]
```

Comment:
Could you mention why this change is needed?

Reply:
Oh sorry, this slipped.
In a docker compose setup with all needed gnosis infrastructure the transaction service would reject requests coming from other containers.
So I've added the default `*` (allow all) and reading from env var `DJANGO_ALLOWED_HOSTS` in case one would like to do something custom. Similar code exists in  https://github.com/gnosis/safe-transaction-service/blob/master/config/settings/production.py#L9

Reply:
Please remove this change from the PR 🙂

Reply:
Can I add it in a separate PR?